rebar.config

```diff
@@ -0,0 +1,52 @@
+{erl_opts,
+ [ debug_info
+ , warn_export_vars
+ , warn_unused_import
+ , warn_missing_spec
+ , warn_untyped_record
+ ]}.
+
+{deps, [{getopt, "1.0.1"}]}.
+
+{escript_incl_apps, [concuerror, getopt]}.
+{escript_main_app, concuerror}.
+{escript_name, concuerror}.
+{escript_emu_args, "%%! +S1 -boot start_clean -noshell -pa . -pa ebin\n"}.
+
+{pre_hooks,
+ [ {compile, "priv/generate_version_hrl 20 > src/concuerror_otp_version.hrl"}
+ ]}.
+
+{post_hooks,
+ [ {escriptize, "cp \"$REBAR_BUILD_DIR/bin/concuerror\" ./bin"}
+ ]}.
+
+{profiles,
+ [ {default,
+    []}
+ , {dev,
+    [ {erl_opts, [{d, 'DEV', "true"}]}
+    ]}
+ , {native,
+    [ {erl_opts, [native]}
+    ]}
+ , {pedantic,
+    [ {erl_opts, [warnings_as_errors]}
+    ]}
+ ]}.
+
+{dialyzer,
+ [ {warnings,
+    [ unmatched_returns
+    , underspecs
+    ]}
+ , {base_plt_apps,
```

Comment:
the apps that are in your .app.src file usually don't need to be repeated here :)

Reply:
Ah yeah, so those end up in the cached PLT rather than re-added dynamically as dependencies of the application every time. I guess that's fine then yeah, since the default is `[erts, crypto, kernel, stdlib]`. Nevermind me then, that's fine :)